responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -474,7 +476,8 @@ def _validate_rai_insights_input_parameters(
         """
         valid_tasks = [
             ModelTask.CLASSIFICATION.value,
-            ModelTask.REGRESSION.value
+            ModelTask.REGRESSION.value,
```

Comment:
we should not be modifying rai_insights - only rai vision insights?

Reply:
We're just modifying rai insights in terms of propagating the OD enum to EA manager and making sure the user input specified for classes is returned as is.